### PR TITLE
Disable pragma that Windows doesn't know about

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

File: libraries/networking/src/DataServerAccountInfo.cpp [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #include "NetworkLogging.h"
 #include "DataServerAccountInfo.h"
 
-#ifndef __GNUC__
+#if !defined(__GNUC__) && !defined(Q_OS_WIN)
```
Comment (sethalves) [inline libraries/networking/src/DataServerAccountInfo.cpp]: This is an improvement, but... is there a way to test of we're compiling with clang?  (full disclosure:  I added the ifndef __GNUC__).

**Comment by hifi-gustavo** [comment]
Build console output for commit: 70f7329a7e804430c3a71833e366fb9394c1486d available at: https://gist.github.com/5475081c189aa2e126a8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4816.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4816.dmg

File: libraries/networking/src/DataServerAccountInfo.cpp (hunk 1 above)
Comment (ctrlaltdavid) [inline libraries/networking/src/DataServerAccountInfo.cpp]: Good question. I'm Windows-only but from Stackoverflow it looks like there might be a __clang__ define: http://stackoverflow.com/questions/28166565/detect-gcc-as-opposed-to-msvc-clang-with-macro/28166605#28166605

Just noticed that there's a similar #pragma in RSAKeypairGenerator.cpp that should be #ifdef'd

**Comment by hifi-gustavo** [comment]
Build console output for commit: 662f07a9214220efc75c4ce3f8740021a122386d available at: https://gist.github.com/5475081c189aa2e126a8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4816.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4816.dmg

**Comment by ctrlaltdavid** [comment]
It looks like the __clang__ #ifdef is probably working but could someone please check it enables the pragma on OSX? Thanks.

**Comment by jherico** [comment]
verified, suppresses deprecation warnings with old SSL libs

**Pull request merged**